wremnants/CardTool.py

```diff
@@ -742,8 +744,11 @@ def loadNominalCard(self):
                 "inputfile" : self.outfile if type(self.outfile) == str  else self.outfile.GetName(),
                 "dataName" : self.dataName,
                 "histName" : self.histName,
-                "pseudodataHist" : self.pseudoData+"_sum" if self.pseudoData else f"{self.histName}_{self.dataName}"
+                "pseudodataHist" : self.pseudoData+"_sum" if self.pseudoData else f"{self.dataName}/{self.histName}"
             }
+            # use the relative path because absolute paths are slow in text2hdf5.py conversion
+            args["inputfile"] = args["inputfile"].split("/")[-1]
```

Comment:
os.path.basename also an option

Reply:
I think the real reason to do this is just so that you can still use the cards after copying to a different location.

Reply:
Yes I think it makes sense, I now also use basename as Kenneth suggested